packages/docs/src/examples/va-sidebar/Color.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="mb-4" style="height: 16rem; border: 2px solid var(--va-primary);">
-    <va-sidebar color="primary" text-color="secondary" minimizedWidth="64px" width="18rem">
+    <va-sidebar color="primary" text-color="white" minimizedWidth="64px" width="18rem">
       <va-sidebar-item>
```

Comment:
```suggestion
      <va-sidebar-item active-color="background">
```

The first sidebar item should also have an active-color prop, like other items. Hover color is relative to active color. You can check colored docs demos and see how different is hover color.